app/assets/scripts/components/share.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 import React from 'react';
 import url from 'url';
+import hideOnBlur from 'react-click-outside';
```

Comment:
Doesn't this library need you to include a `handleClickedOutside` method? [link](https://github.com/kentor/react-click-outside#usage)